UM/View/GL/OpenGL.py

```diff
@@ -19,31 +32,68 @@ class Vendor:
         Intel = 3
         Other = 4
 
+    def __init__(self):
+        profile = QOpenGLVersionProfile()
```

Comment:
What do you mean? The profile object is created to get versionFunctions. The Qt API is somewhat strange here IMHO.